ipsie-levels.md

```diff
@@ -1,85 +1,60 @@
 # IPSIE Levels
 
-
+- *A* - Authentication
 - *SL* - Session Lifecycle
 - *IL* - Identity Lifecycle
-
-| IPSIE<br>LEVEL|   Application<br>(aka RP)                             |  Enterprise<br>(aka IdP)                         |
-|---------------|-------------------------------------------------------|---------------------------------------------------|
-| SL1           |  - FAL2<br>- Session lifetime MUST match assertion lifetime  | - MFA enforced and sent to App  |
-| SL2           |  - Requests MFA<br>- MUST accept session termination request | - MUST accept MFA level requests |
-| SL2           |  - MUST communicate posture changes                   | - MUST communicate posture changes                |
-|               |                                                       |                                                   |
-| IL1           |  - MUST support JIT and MUST accept attributes        | - MUST send centrally managed attributes            |
-| IL2           |  - MUST support mapping group claims to roles         | - MUST send selected group claims                   |
-| IL3           |  - MUST support accepting async identity lifecycle requests    | - MUST support sending identity lifecycle requests |
-
-Each level includes the previous level (eg IL3 includes IL2 and IL1)
-
+- *E* - Entitlements
+
+Each level includes the previous level (_e.g._ SL3 includes the requirements of SL1 and SL2). Requirements at a lower level may be made OPTIONAL at a higher level if explicitly defined. (e.g. JIT provisioning is required at IL1 and OPTIONAL at IL2). Each set of levels is _independent_ from other levels (e.g. an application may achieve E3 while all other sets are at Level 1).
```

Comment:
```suggestion
Each level includes the previous level (_e.g._ SL3 includes the requirements of SL1 and SL2). Each set of levels is _independent_ from other levels (e.g. an application may achieve E3 while all other sets are at Level 1).
```

I don't think it's a good idea to let higher levels make things at lower levels optional. That sounds like a nightmare of an interoperability matrix to test.